examples/mysql-wordpress-pd/README.md

```diff
@@ -34,301 +34,231 @@ Documentation for other releases can be found at
-3. Install the [gcloud SDK](https://cloud.google.com/sdk/).
-
-Authenticate with gcloud and set the gcloud default project name to point to the project you want to use for your Kubernetes cluster:
-
-```sh
-gcloud auth login
-gcloud config set project <project-name>
+This example describes how to run a persistent installation of
+[WordPress](https://wordpress.org/) on Kubernetes, using
+[Persistent Volumes](../../docs/user-guide/persistent-volumes.md) and
+[Secrets](../../docs/user-guide/secrets.md).
+
+We'll use the [mysql](https://registry.hub.docker.com/_/mysql/) and
+[wordpress](https://registry.hub.docker.com/_/wordpress/) official
+[Docker](https://www.docker.com/) images for this installation. (The
+wordpress image includes an Apache server). We'll create two Kubernetes
+[Deployments](../../docs/user-guide/deployments.md) to run MySQL and
+WordPress, both with associated persistent volmes and
+[Services](../../docs/user-guide/services.md) to front each
+deployment.
+
+This example demonstrates several useful things, including: how to set
+up and use persistent disks with Kubernetes pods; how to define
+Kubernetes services to enable pods to find each other; and use of an
+external load balancer to expose the WordPress service externally and
+make it transparent to the user if the WordPress pod moves to a
+different cluster node.
+
+## tl;dr Quickstart
+
+Put your desired mysql password in a file called `password.txt` with no
+trailing newline.
+
+```shell
+kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/mysql-wordpress-pd/local-volumes.yaml
+kubectl create secret generic mysql-pass --from-file=password.txt
+kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/mysql-wordpress-pd/mysql-deployment.yaml
+kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/mysql-wordpress-pd/wordpress-deployment.yaml
 ```
 
-Next, start up a Kubernetes cluster:
-
-```sh
-wget -q -O - https://get.k8s.io | bash
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+##Table of Contents
+
+  - [Cluster Requirements](#cluster-requirements)
+  - [Decide where you will store your data](#decide-where-you-will-store-your-data)
+    - [GCE Persistent Disk](#gce-persistent-disk)
+    - [Host Path](#host-path)
+  - [Create the MySQL Password Secret](#create-the-mysql-password-secret)
+  - [Deploy MySQL](#deploy-mysql)
+  - [Deploy WordPress](#deploy-wordpress)
+  - [Visit your new WordPress blog](#visit-your-new-wordpress-blog)
+  - [Take down and restart your blog](#take-down-and-restart-your-blog)
+  - [Next Steps](#next-steps)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Cluster Requirements
+
+Consult a
+[Getting Started Guide](../../docs/getting-started-guides/)
```

Comment:
I think the getting-started-guides README needs an overhaul to highlight certain deployments. I'll update this link to link directly to the README, so there isn't the whole directory listing to scroll past.

For the examples, we don't want to have cluster start up instructions in each one, this should be covered by this link. Ideally each example lists the requirements, and doesn't cover all the possible environments that will work. I think I'll change this to list the requirements first, and then link to the getting-started-guides after.

For examples that use PVs, we want to change them all to use PVCs, and then add a requirement that a PV be available to be claimed. Thus removing any specific cloud provider dependencies.

Maybe I'll take out both the GCE and Host Path sections, and only link to here: https://github.com/kubernetes/kubernetes/tree/master/docs/user-guide/persistent-volumes where we will need to have these documented.